erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -162,7 +162,7 @@ def add_items(self, items):
 				'include_exploded_items': 1,
 				'warehouse': data.warehouse,
 				'item_code': data.item_code,
-				'description': item_details and item_details.description or '',
+				'description': data.description,
```

Comment:
data.description or item_details.description?

Reply:
For me data.description is better.
data.description will fetch description from sales order (items / packed items) and from material requests, item_details.description will fetch it from item_masters
If a vendor modify description in sales order or in packed items, or in material request, i think it is better for manufacture to see it over the production plan and work orders.
In the same logic than https://github.com/frappe/erpnext/pull/18907
Sincerely yours,
Martin

Reply:
Agreed, but in case if description is blank(no in your case) then system should have to set the description from the item master.

```
'description': data.description or item_details.description
```

Reply:
done